config/initializers/devise.rb

```diff
@@ -0,0 +1,313 @@
+# frozen_string_literal: true
+
+# Assuming you have not yet modified this file, each configuration option below
+# is set to its default value. Note that some are commented out while others
+# are not: uncommented lines are intended to protect your configuration from
+# breaking changes in upgrades (i.e., in the event that future versions of
+# Devise change the default values for those options).
+#
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '260d6e71f8e40da4736acaa8e45311a3ccd96d0db202d8db0218e6afaf2cbe660a940902375b9fee57a8f2f45af598c7e96b0b35fc70cda4b1d65d3746d81b54'
```

Comment:
```suggestion
  # config.secret_key = ''
```
Let's remove this so it doesn't get flagged by any security scanning software as leaking a secret key.

Reply:
Sounds good, I'll do that!